src/js/create.js

```diff
@@ -0,0 +1,105 @@
+function headerComponent() {
+  const header = document.createElement('header');
+  // refresh button
+  const button = document.createElement('button');
+  button.classList.add('btn', 'refresh');
+  button.innerHTML = '<i>&#x21bb;</i>';
+  header.appendChild(button);
+  // language dropdown
+  const dropdown = document.createElement('div');
+  dropdown.setAttribute('class', 'dropdown');
+  dropdown.innerHTML = `
+    <button class="btn dropdown-toggle" type="button" id="dropdownMenuButton1" data-bs-toggle="dropdown" aria-expanded="false">
+      EN
+    </button>
+    <ul class="dropdown-menu" aria-labelledby="dropdownMenuButton1">
+      <li><a class="dropdown-item" href="#">RU</a></li>
+      <li><a class="dropdown-item" href="#">PL</a></li>
+    </ul>`;
+  header.appendChild(dropdown);
```

Comment:
same as above - could be a separate function `createDropDown`

Reply:
done)